### PR TITLE
Fix createsuperuser

### PR DESCRIPTION
**Pull request by tdelatorre**
Fix a bug in createsuperuser. Now you can do: python manage.py createsuperuser

### CODE REVIEW

File: taiga/users/models.py [hunk 1]
```diff
@@ -116,10 +115,9 @@ def create_superuser(self, username, email, password):
         user = self.create_user(
             username=username,
             email=email,
-            password=password
+            is_superuser=True
         )
-        user.is_superuser = True
-        user.is_admin = True
+        user.set_password(password)
```
Comment (yamila-moreno) [inline taiga/users/models.py:120]: set_password is already done in self.create_user

File: taiga/users/models.py [hunk 2]
```diff
@@ -116,10 +115,9 @@ def create_superuser(self, username, email, password):
...
         user.save()
```
Comment (yamila-moreno) [inline taiga/users/models.py:121]: this .save is already done in create_user, should be removed from here

Comment (tdelatorre) [reply taiga/users/models.py]: Done!

Comment (tdelatorre) [reply taiga/users/models.py]: Done!

**Pull request merged**